data/hai/unfettered 2 war preparations.txt

```diff
@@ -0,0 +1,2164 @@
+					goto agree
+				`	"Is it common for the Unfettered to use heralds to bring messages?"`
+			`	"We don't believe technical ways of communicating to be safe, and they break the link between us. Messages could easily be spied upon by anyone intercepting it, so for some confidential missions like this one, it is quite common to use heralds. On top of that, the warlords generally have better things to do than moving around giving missions all the time. Do you have enough information? What shall I say to my warlord?"`
+			choice
+				`	"I will depart as soon as I can."`
+					goto agree
+				`	"You could have just sent a message on a secure channel."`
+			`	"What is secure? There is no code that cannot be broken, and I did not have to travel far anyway. The Fettered Hai have become quite apt at languages, with their interest to communicate with other species like the Quarg or humans. Encryption is just like another language. We cannot risk them learning of this particular mission - you will understand why later. Information is the key here, and it always has been. We also don't want those up north to learn about this. They would surely be able to decrypt it.`
+			`	"We are not all as brutal as we let other races think. My warlord believes that it is to our advantage to not appear cunning, so that they may underestimate us." He seems to have had enough of talking, and is now waiting for you to answer He-Yi's request.`
+			choice
+				`	"I will depart as soon as I can."`
+				`	"This seems too risky for me."`
+					decline
+			label agree
+			`	"Good! You have two months to get the item and bring it back, use them well. Oh, and I think He-Yi would be most impressed if you manage to acquire the item without looting it off the enemy ship."`
+				accept
+	on waypoint
+		dialog
+			`It seems that the <npc> is in this system, and a Syndicate attack just launched!`
+			`Judging by the marks on its hull, it will likely land to make repairs and restock ammo soon. You may be able to get the item without too much risk that way, and bribing them will mean secure landing. Or you could always just board it now and be done with it. You could even sabotage the ship to make your job easier, or take some more valuables with you.`
+	# note: this only works because the fleets do not fight if you are not in the system.
+	# If that was to change we'd need a temporary fleet with an event changing the reputation.
+	npc
+		personality coward plunders vindictive staying
+		government "Mhairi's Legion"
+		system "Alcyone"
+		fleet "Large Northern Pirates" 3
+		fleet "Small Northern Pirates" 5
+	npc
+		personality derelict uninterested coward
+		government "Mhairi's Legion"
+		system "Alcyone"
+		fleet "Large Northern Pirates"
+		fleet "Small Northern Pirates"
+	npc
+		personality heroic staying entering disables
+		to despawn
+			has "<uhai> 17: Spy On Hai B: offered"
+		government "Syndicate"
+		system "Alcyone"
+		fleet "Large Syndicate" 3
```

Comment:
The Syndicate will be annihilated here, so should this number be higher?

Reply:
they're meant to get rekt, I dont like fleet battles that randomly decide an outcome

Reply:
well mb I should bring it up a bit but then smbd would have to test it

Reply:
No, you’re right. I’m just thinking from the Syndicate’s point of view, "We’ll send some of our big warships. Three to six is enough. They got annihilated?! Too bad, other things to do." I’m not sure quite makes sense. Of course, they’re all about profit, but…